core/worker.go

```diff
@@ -243,13 +243,6 @@ func newWorker(config *Config, chainConfig *params.ChainConfig, db ethdb.Databas
 	// Subscribe events for blockchain
 	worker.chainSideSub = headerchain.bc.SubscribeChainSideEvent(worker.chainSideCh)
 
-	// Sanitize recommit interval if the user-specified one is too short.
-	recommit := worker.config.Recommit
-	if recommit < minRecommitInterval {
```

Comment:
Why did you remove this? I feel like we should still saturate the recommit interval to some minimum interval, no?

Reply:
This is done in the `GeneratePendingHeader`. This was a copy only geting triggered on the initialization of the worker.